source/common/umf_pools/disjoint_pool.hpp

```diff
@@ -46,7 +46,7 @@ class DisjointPoolConfig {
 
     // Holds the minimum bucket size valid for allocation of a memory type.
     // This value must be a power of 2.
-    size_t MinBucketSize = 1;
+    size_t MinBucketSize = 8;
```

Comment:
make an a const global variable for this value (and use it here and in the implementation).

Reply:
Done.